googletest/include/gtest/internal/gtest-param-util-generated.h

```diff
@@ -79,6 +79,8 @@ class ValueArray1 {
     return ValuesIn(array);
   }
 
+  ValueArray1(const ValueArray1& other) : v1_(other.v1_) {}
```

Comment:
I haven't looked into what this `ValueArray` does, so I could easily be wrong, but couldn't this line just be:
```
ValueArray1(const ValueArray1&) = default;
```
because this explicit copy constructor sure looks like what `= default;` would generate from the compiler. In which case every other change in this commit could also be a single `= default;` line, and not have to explicitly copy each member of `other` into `*this` based n the particular `ValueArray`. No?

Reply:
It is a C++11 feature but GoogleTest keeps compatibility with C++03.